test/controllers/deceased_settlor/SettlorDateOfDeathYesNoControllerSpec.scala

```diff
@@ -26,8 +26,7 @@ import views.html.deceased_settlor.SettlorDateOfDeathYesNoView
 
 class SettlorDateOfDeathYesNoControllerSpec extends SpecBase {
 
-  val formProvider = new SettlorDateOfDeathYesNoFormProvider()
-  val form = formProvider()
+  val form = new YesNoFormProvider().withPrefix("settlorDateOfDeath")
```

Comment:
Should prefix not  be "settlorDateOfDeathYesNo"?

Reply:
Complete